demo/index.html

```diff
@@ -1,81 +1,758 @@
+                        <path d="M4 8l16 0" />
+                        <path d="M4 16l16 0" />
+                    </svg>
+                </button>
+            </header>
+            <div id="navigation-menu">
+                <div id="background-overlay"></div>
+                <div id="menu">
+                    <div id="menu-links">
+                        <header>
+                            <span>Menu</span>
+                            <button id="close-button">
+                                <svg
+                                    xmlns="http://www.w3.org/2000/svg"
+                                    width="24"
+                                    height="24"
+                                    viewBox="0 0 24 24"
+                                    fill="none"
+                                    stroke="currentColor"
+                                    stroke-width="2"
+                                    stroke-linecap="round"
+                                    stroke-linejoin="round"
+                                    class="icon icon-tabler icons-tabler-outline icon-tabler-x"
+                                >
+                                    <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                                    <path d="M18 6l-12 12" />
+                                    <path d="M6 6l12 12" />
+                                </svg>
+                            </button>
+                        </header>
+                        <menu>
+                            <a href="#introduction">Introduction</a>
+                            <a href="#why">Why</a>
+                            <a href="#installation">Installation</a>
+                            <div class="menu-indented">
+                                <a href="#download-the-polyfill">Download the polyfill</a>
+                                <a href="#cdn-url">Via a CDN url</a>
+                                <a href="#esm">Via ESM</a>
+                            </div>
+                            <a href="#polyfill-behaviour">Polyfill behaviour</a>
+                            <a href="#overriding-default-behaviour"
+                                >Overriding the default behaviour</a
+                            >
+                            <a href="#examples">Examples</a>
+                            <div class="menu-indented">
+                                <a href="#dragging-some-boxes">Dragging some boxes</a>
+                                <a href="#sorting-tasks">Sorting tasks</a>
+                                <a href="#kanban-board">Kanban board</a>
+                            </div>
+                        </menu>
+                    </div>
+                    <span class="copyright"
+                        >&copy; <span class="current-year"></span>, drag-drop-touch-js</span
+                    >
+                </div>
+            </div>
+            <div id="content">
```

Comment:
We don't need to duplicate the entire nav, just use `@media (...)` on the CSS side to style things appropriately based on viewport dimensions.